src/main/java/com/hubspot/jinjava/lib/filter/IndentFilter.java

```diff
@@ -40,24 +39,22 @@
     )
   }
 )
-public class IndentFilter implements Filter {
+public class IndentFilter extends AbstractFilter {
 
   @Override
   public String getName() {
     return "indent";
   }
 
   @Override
-  public Object filter(Object var, JinjavaInterpreter interpreter, String... args) {
-    int width = 4;
-    if (args.length > 0) {
-      width = NumberUtils.toInt(args[0], 4);
-    }
+  public Object filter(
+    Object var,
+    JinjavaInterpreter interpreter,
+    Map<String, Object> parsedArgs
+  ) {
+    int width = ((Number) parsedArgs.get("width")).intValue();
 
-    boolean indentFirst = false;
-    if (args.length > 1) {
-      indentFirst = BooleanUtils.toBoolean(args[1]);
-    }
+    boolean indentFirst = (boolean) parsedArgs.get("indentfirst");
```

Comment:
could you create a static constant for "indentfirst" and "width"?

Reply:
Done